### ISSUE TITLE
Lazy load don't work if gradient is defined on the image 

### ISSUE THREAD

**Issue by ajays1991** [tuupola/jquery_lazyload#370]
Hi,

I found out that lazy load don't actually lazily load the images on the html page if images in the html has gradient defined. We checked the source code and forked the repository to fix/workaround as our internal solution. I thought of letting you know.

Thanks

**Comment by tuupola** [comment]
What was the fix? How does the html or css look when the problem occurs?

**Comment by ajays1991** [comment]
yeah, it was me who fixed did the workaround. i implemented a very quick fix to get this working as we were being thrown back by investors/gmeter on SEO grounds for page size.
It was forked in org repo.
I will spare some time and i can atleast point out the code around where i did the fix. 

Thanks
